foundation/__init__.py

```diff
@@ -13,9 +13,14 @@ def get_last_membership():
 
 	return last_membership and last_membership[0]
 
+def get_all_memberships_of_one_member():
+	'''Returns last membership if exists'''
+	all_memberships = frappe.get_all("Membership", fields=["name", "from_date", "to_date", "membership_type", "amount","currency"], filters={"member":"asbasawaraj@gmail.com","paid": "1"},order_by="to_date desc")
```

Comment:
Looks like this function needs a `member` argument, for the value of the hardcoded filter :)